source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md

```diff
@@ -0,0 +1,65 @@
+---
+title: vagrant-service-manager 1.3.0 released focusing on Kubernetes support
+author: budhram
+date: 2016-08-16 12:54:00 UTC
+tags: vagrant, devtools, releases, kubernetes
+published: true
+comments: true
+---
+
+This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes setup information which enable users to access Kubernetes server running inside ADB VM from host.
+
+We also extended the client binary installation support for 'Kubernetes' in this release. Not only that, users now can install these client binaries in [CDK (Container Development Kit)](http://developers.redhat.com/products/cdk/overview/) platform too. The overview of the client binary installation feature can be found in the previous blog ['Client Binary Installation Now Included in the ADB'](blog/2016/07/vagrant-service-manager-install-cli).
+
+The main features of this release are:
+
+* Enable displaying of Kubernetes setup information so one can access Kubernetes running inside ADB VM
+* Client binary installation support for Docker, OpenShift CDK
```

Comment:
* Client binary installation support for OpenShift, Kubernetes, and docker in the Red Hat Container Development Kit